src/graph/entities/graph_entity.c

```
@@ -67,16 +67,18 @@
 ) {
-	GraphContext *gc = QueryCtx_GetGraphCtx();
-	const AttributeSet set = GraphEntity_GetAttributes(e);
-	int propCount = AttributeSet_Count(set);
-	SIValue map = SI_Map(propCount);
+	GraphContext      *gc        = QueryCtx_GetGraphCtx();
+	const AttributeSet set       = GraphEntity_GetAttributes(e);
+	int                propCount = AttributeSet_Count(set);
+	SIValue            map       = SI_Map(propCount);
+
 	for(int i = 0; i < propCount; i++) {
 		AttributeID attr_id;
```

cppcheck (2.10-2)

[error] Memory leak

(memleak)